src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,47 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
-        List<Product> products = getAllProductsFromShoppingCart(userId);
+        logger.info("completeOrder method was called for userId: {}", userId);
+        List<Product> products = null;
+        try {
+            products = getAllProductsFromShoppingCart(userId);
+            logger.debug("Products retrieved from shopping cart for userId: {}: {}",
+                    userId, products);
+        } catch (Exception e) {
+            logger.error("Failed to retrieve products from shopping cart for userId: {}",
+                    userId, e);
+        }
+
         Order order = new Order(products, userId);
+        logger.info("Order created for userId: {} with products: {}", userId, products);
```

Comment:
The log message when creating an order is good, but it might be more appropriate as a debug log rather than info since it seems to be part of the normal operation flow and may not need to be logged at the info level.